src/routes/Profile/Competences/loader.ts

```diff
@@ -19,6 +21,9 @@ export default (async ({ request }) => {
   const { data: vapaaAjanToiminnot } = await client.GET('/api/profiili/vapaa-ajan-toiminnot', {
     signal: request.signal,
   });
+  const muutOsaamiset = await client
+    .GET('/api/profiili/muu-osaaminen', { signal: request.signal })
+    .then((res) => osaamisetService.find(res.data));
```

Comment:
Fetching the various "osaaminen" could be done in parallel (`await Promise.all(...)`). There is also some redundancy here: `/api/profiili/osaamiset` already returns all osaaminen (including muu osaaminen) so it is not necessary to fetch it separately ( `{ ...,  "lahde": { "tyyppi": "MUU_OSAAMINEN" }`).


```
{
	"11": {
		"id": "ff3c2b61-83ce-4b52-b96e-25ef9d1d6812",
		"osaaminen": {
			"uri": "http://data.europa.eu/esco/skill/21c5790c-0930-4d74-b3b0-84caf5af12ea",
			"nimi": {
				"en": "manage budgets",
				"fi": "hallita budjetteja",
				"sv": "administrera budgetar"
			},
			"kuvaus": {
				"en": "Plan, monitor and report on the budget.",
				"fi": "Budjettien suunnittelu, seuraus ja niistä raportointi.",
				"sv": "Planera, granska och rapportera om budgeten."
			}
		},
		"lahde": {
			"tyyppi": "MUU_OSAAMINEN"
		}
	}
}
```

Reply:
Fixed!